src/workloads/execution/validate.yml

```diff
@@ -0,0 +1,36 @@
+# This workload inserts 1GB of documents and runs validate()
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 1e4  # TODO: increase the document size/count so there is more data.
```

Comment:
TODO still valid?

Reply:
Yep. This is meant to be a template workload for Gregory to fill in based on his requirements